imperial.test.js

```diff
@@ -0,0 +1,113 @@
+import Imperial from "./imperial";
+import { Nation } from "./constants.js";
+
+describe("Imperial", () => {
+  describe("constructor", () => {
+    describe("six players", () => {
+      const state = {
+        players: [
+          { id: "a", nation: Nation.RU },
+          { id: "b", nation: Nation.FR },
+          { id: "c", nation: Nation.GB },
+          { id: "d", nation: Nation.AH },
+          { id: "e", nation: Nation.IT },
+          { id: "f", nation: Nation.GE },
+        ],
+      };
+      const sut = new Imperial(state);
```

Comment:
What is `sut`?

Reply:
System Under Test

but i think that's probably more confusing than `game`